src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -0,0 +1,359 @@
+            gbl_panelContent.rowWeights = new double[] { 1.0, Double.MIN_VALUE };
+            panelContent.setLayout(gbl_panelContent);
+            GridBagConstraints gbc_scrollPaneInventory = new GridBagConstraints();
+            gbc_scrollPaneInventory.fill = GridBagConstraints.BOTH;
+            gbc_scrollPaneInventory.gridx = 0;
+            gbc_scrollPaneInventory.gridy = 0;
+            panelContent.add(getScrollPaneInventory(), gbc_scrollPaneInventory);
+        }
+        return panelContent;
+    }
+
+    private JPanel getPanelFooter() {
+        if (panelFooter == null) {
+            panelFooter = new JPanel();
+            panelFooter.add(getNewButton());
+            panelFooter.add(getViewButton());
+            panelFooter.add(getUpdateButton());
+            panelFooter.add(getDeleteButton());
+            panelFooter.add(getCloseButton());
+        }
+        return panelFooter;
+    }
+
+    private GoodDateChooser getJCalendarFrom() {
+        if (jCalendarFrom == null) {
+            jCalendarFrom = new GoodDateChooser(dateFrom.toLocalDate());
+            jCalendarFrom.addDateChangeListener(dateChangeEvent -> {
+                LocalDate newDate = dateChangeEvent.getNewDate();
+                if (newDate != null) {
+                    dateFrom = newDate.atStartOfDay();
+                    InventoryBrowsingModel inventoryModel = new InventoryBrowsingModel();
+                }
+            });
+            jCalendarFrom.setEnabled(false);
+        }
+        return jCalendarFrom;
+    }
+
+    private GoodDateChooser getJCalendarTo() {
+        if (jCalendarTo == null) {
+            jCalendarTo = new GoodDateChooser(dateTo.toLocalDate(), false);
+            jCalendarTo.addDateChangeListener(dateChangeEvent -> {
+                LocalDate newDate = dateChangeEvent.getNewDate();
+                if (newDate != null) {
+                    dateTo = newDate.atStartOfDay();
+                    InventoryBrowsingModel inventoryModel = new InventoryBrowsingModel();
+                }
+            });
+            jCalendarTo.setEnabled(false);
+        }
+        return jCalendarTo;
+    }
+
+    private JLabel getJLabelTo() {
+        if (jLabelTo == null) {
+            jLabelTo = new JLabel();
+            jLabelTo.setHorizontalAlignment(SwingConstants.RIGHT);
+            jLabelTo.setText(MessageBundle.getMessage("angal.common.to.txt")); //$NON-NLS-1$
+        }
+        return jLabelTo;
+    }
+
+    private JLabel getJLabelFrom() {
+        if (jLabelFrom == null) {
+            jLabelFrom = new JLabel();
+            jLabelFrom.setHorizontalAlignment(SwingConstants.RIGHT);
+            jLabelFrom.setText(MessageBundle.getMessage("angal.common.from.txt")); //$NON-NLS-1$
+        }
+        return jLabelFrom;
+    }
+
+    private JButton getNewButton() {
+        newButton = new JButton(MessageBundle.getMessage("angal.inventory.new.btn"));
+        newButton.setMnemonic(KeyEvent.VK_N);
```

Comment:
You can't hardcode a mnemonic because the text of the button will change with the language.   Review other entries in the bundle to see how it is done.